amundsen_application/wsgi.py

```diff
@@ -1,9 +1,9 @@
 import os
 from amundsen_application import create_app
 
-app = create_app(
+application = create_app(
```

Comment:
In what situation did gunicorn output `NameError: name 'application' is not defined`? I understand you experienced some error and that this fixed it, but I'm not understanding why that error occurred.

Reply:
It happens on startup of the application. I didn’t explore further when I implemented, just followed instructions in Gunicorn’s error message. But I now see they default to looking for `”application”` in `import_app()` here: https://github.com/benoitc/gunicorn/blob/19.9.0/gunicorn/util.py#L362

I’m more inclined to just follow the simple default case if we don’t have anything else depending on it being called `app` and not the default (I was actually wondering why the frontend  name was different than the other Flask services). 

If needed it looks like they take a `:` separated module name parameter, so should be easy to have Gunicorn work with `app` though.

Reply:
Not sure I quite understand, but it's alright. I'll read up more on it.